script.js

```diff
@@ -67,12 +67,34 @@ function validateEmail() {
   return true;
 }
 
+// LOCAL SRORAGE
+const formName = document.getElementById('user-name');
+const email = document.getElementById('user-email');
+const formText = document.getElementById('user-text');
+
+const localData = () => {
+  const userData = {
+    userName: formName.value,
+    UserMail: email.value,
+    txt: formText.value,
+  };
+  localStorage.setItem('userData', JSON.stringify(userData));
+};
+const storedDetails = () => {
+  const info = JSON.parse(localStorage.getItem('userData'));
+  formName.value = info.formName;
+  email.value = info.email;
+  formText.value = info.formText;
+};
 function validateForm() {
   if (!validate() || !validateEmail()) {
     submitError.style.display = 'block';
     submitError.innerHTML = 'Please fix errors to submit';
     setTimeout(() => { submitError.style.display = 'none'; }, 3000);
     return false;
-  } return true;
+  }
```

Comment:
Great job on using the required data model however in this milestone you have to utilise local storage to persist data in your form and according to the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/10_localstorage.md#project-requirements), you should implement the following interactions:

- When the user changes the content of any input field, the data is saved to the local storage.
- When the user loads the page, if there is any data in the local storage the input fields are pre-filled with this data.

- [ ] Currently no data is being saved to the local storage (You can check it using developer tools:  inspect   => application => localStorage). Please make sure when the user inputs something in the input field the data gets stored to the localStorage. And if the user reloads the page the data should remain in the input field. Example snippet:

window.onload = () => {
  let savedFormData = localStorage.getItem('contactFormInput');
  savedFormData = JSON.parse(savedFormData);
  // Check if the form data object is found on localStorage
  if (savedFormData) {
  // populate inputs values if data was found
  // ex: nameInput.value = savedFormData.name
  }
};